scripts/monitoring/ops-metric-client.py

```diff
@@ -64,6 +64,14 @@ def run(self):
 
         self.metric_sender.send_metrics()
 
+    @staticmethod
+    def adjust_type(val):
+        """ convert float values to float type. keep string as is """
+        try:
+            return float(val)
```

Comment:
I can add a cli parameter to indicate metric type. 
Problem is what will be the default? for backward compatibility with existing jobs?